### ISSUE TITLE
These labels still exist in annotations error

### ISSUE THREAD

**Issue by serjrd** [heartexlabs/label-studio#734]
Getting `These labels still exist in annotations` error when trying to delete object classes in labeling interface even though all objects of this class were removed.

Tried using label-studio filter by "annotations results" and also made a direct SQL-query `select * from task_completion tc where "result" LIKE '%cage%'` to check that there are no more annotations of that class.

Restarting label-studio doesn't help.

**Environment:**
 - OS: Ubuntu 20.04
 - Label Studio Version 1.0.0-post3

**Comment by makseq** [comment]
@serjrd could you check release/1.0.1 branch? Will it persist there?

**Comment by serjrd** [comment]
@makseq

Yes it does

**Comment by niklub** [comment]
Hi, @serjrd ! 
Do you have any clues on how to reproduce this issue in a newly created project?

**Comment by csaroff** [comment]
I'm running into this issue as well

**Comment by makseq** [comment]
Please check label-studio==1.4.1rc0

**Comment by csaroff** [comment]
This appears to be fixed as of v1.4.1post1(didn't try rc0).

Thanks @makseq!

**Comment by hyp530** [comment]
Had same issue today with latest version.

**Comment by john-** [comment]
I just had this same issue as well.  I did a dump of the postgres database and did a grep for the classes listed in the error.  They were all in the `projects_projectsummary` table.  Some appeared in the table that tracks my filters but those were just part of debugging the problem.

I used psql to delete the single row in `projects_projectsummary`.  After stopping the docker containers for Label Studio and starting it the error went away.  Also, there was a single row added back to `projects_projectsummary` that showed all clear.

I have only done basic confirmation that things are working.  I would have been more careful if the occurrences were in other tables as well.  This may break things in a way I have not seen yet but in my case situation I would just start over.
